### PR TITLE
Use Tailwind CSS for better reactive styles

### PR DESCRIPTION
**Pull request by canadianveggie**
Needs a lot of work still, but it leads to some nicer resizing behaviour.

To compile the CSS:
`npx tailwindcss -i ./public/input.css -o ./public/main.css --watch`

### CODE REVIEW

**Comment by canadianveggie** [comment]
Need to figure out a clean way to incorporate tailwind compilation (either as a github action or a precommit hook)

File: public/main.css [hunk 1]
```diff
@@ -1,3 +1,549 @@
...
+  resize: vertical;
+}
+
+/*
+1. Reset the default placeholder opacity in Firefox. (https://github.com/tailwindlabs/tailwindcss/issues/3300)
+2. Set the default placeholder color to the user's configured gray 400 color.
+*/
+
+input::-moz-placeholder,
+textarea::-moz-placeholder {
```
Comment (wachunga) [inline public/main.css:164]: there are a lot of resets here we don't need, but I guess it's fine

File: public/main.css [hunk 2]
```diff
@@ -321,49 +868,103 @@ a {
...
+  -webkit-animation: pop 0.3s;
   animation: pop 0.3s;
 }
+
 .fadein {
+  -webkit-animation: fadein 0.5s;
   animation: fadein 0.5s;
 }
 
+@-webkit-keyframes pulse {
```
Comment (wachunga) [inline public/main.css:574]: this autoprefixing probably isn't necessary in 2022 but I guess it doesn't hurt much

File: index.html [hunk 3]
```diff
@@ -122,7 +124,7 @@
                     ></span>
                   </div>
                 </template>
-                <div class="guesses-tile emoji-tile">
+                <div class="guesses-tile emoji-tile basis-1/6">
```
Comment (wachunga) [inline index.html:127]: this basically just sets width to 16%... which maybe works well enough now that we're using CSS grid etc

File: index.html [hunk 4]
```diff
@@ -68,8 +67,8 @@
           </a>
         </div>
       </header>
-      <div class="game-content" x-show="page === ''">
-        <div class="radio-toolbar">
+      <div class="game-content items-center" x-show="page === ''">
+        <div class="radio-toolbar items-center">
```
Comment (wachunga) [inline index.html:71]: if we're going to switch to tailwind, we should transition away these semantic classnames eventually. The idea is to not write CSS yourself.

**Pull request closed without merging**